docs/src/main/sphinx/release/release-430.md

```diff
@@ -0,0 +1,47 @@
+# Release 430 (20 Oct 2023)
+
+## General
+
+* Improve performance of freshness calculation for materialized views. ({issue}`19423`)
+* Fix ordering of returned result set when query has a window function with
+  identical ordering. ({issue}`19399`)
+* Fix incorrect results for query involving an aggregation in a correlated
+  subquery. ({issue}`19002`)
+* Fix `RunningQueries` metric to represent all running queries. ({issue}`19427`)
+
+## Security
+
+* Enforce authorization capability of client when receiving commands `RESET` and
+  `SET` for `SESSION AUTHORIZIATION`. ({issue}`19217`)
+* Switch error code to `TABLE_NOT_FOUND` if all columns filtered out from access
```

Comment:
This is a low level implementation detail. What's the user-visible impact?

Reply:
I think that error bubbles up ... also I am not convinced this change is right anyway

Reply:
Leaving out.